doc/DeveloperGuide.md

```diff
@@ -87,6 +87,35 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+  
+#### Use case: Rename tags
+
+**MSS**
+
+1. User requests to rename tag.
+2. AddressBook request the tag to be changed and the new tag.
+3. User specifies the tag to be changed and the new tag.
+4. AddressBook request for confirmation (yes/no) of change.
```

Comment:
Generally in use cases we want to specify the intention, so "requesting for confirmation" is sufficient.